modules/assim.sequential/R/downscale_function.R

```diff
@@ -61,25 +57,25 @@ NA_downscale <- function(data, cords, covariates, focus_year, C_pool){
   }
   
   # Train a random forest model for each ensemble member using the training data
-  output <- list()
+  rf_output <- list()
   for (i in 1:length(ensembles)) {
-    output[[i]] <- randomForest::randomForest(ensembles[[i]][[1]][["carbon_data"]] ~ land_cover+tavg+prec+srad+vapr+nitrogen+phh2o+soc+sand,
-                                data = ensembles[[i]][[1]],
-                                ntree = 1000,
-                                na.action = stats::na.omit,
-                                keep.forest = T,
-                                importance = T)
+    rf_output[[i]] <- randomForest::randomForest(ensembles[[i]][[1]][["carbon_data"]] ~ land_cover+tavg+prec+srad+vapr+nitrogen+phh2o+soc+sand,
+                                                 data = ensembles[[i]][[1]],
+                                                 ntree = 1000,
+                                                 na.action = na.omit,
```

Comment:
why has `stats::na.omit` been reverted to `na.omit`, this will lead to a GH Actions error

Reply:
```suggestion
                                                 na.action = stats::na.omit,
```